### PR TITLE
Sort the benchmarks before listing them

### PR DESCRIPTION
**Pull request by ggwpez**

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: substrate/utils/frame/benchmarking-cli/src/pallet/command.rs [hunk 1]
```diff
@@ -748,13 +748,16 @@ impl CliConfiguration for PalletCmd {
...
-	benchmarks_to_run: Vec<(
+	mut benchmarks_to_run: Vec<(
 		Vec<u8>,
 		Vec<u8>,
 		Vec<(BenchmarkParameter, u32, u32)>,
 		Vec<(String, String)>,
 	)>,
 ) {
+	// Sort by pallet and function name.
+	benchmarks_to_run.sort_by(|(pa, sa, _, _), (pb, sb, _, _)| (pa, sa).cmp(&(pb, sb)));
```
Comment (bkontur) [inline substrate/utils/frame/benchmarking-cli/src/pallet/command.rs:759]: What about multiple pallet instances e.g. pallet_assets?
Shouldnt we consider also `BenchmarkList.instance` attribute?

Comment (ggwpez) [reply substrate/utils/frame/benchmarking-cli/src/pallet/command.rs:759]: It is currently not possible to benchmark only a single instance of a pallet. And IMHO it does not make much sense.  
But they will be listed multiple times, i will add a dedub to it.

Comment (bkontur) [reply substrate/utils/frame/benchmarking-cli/src/pallet/command.rs:759]: ok then,
I know the only one place where `list` command is used: https://github.com/paritytech/command-bot-scripts/blob/main/commands/bench/lib/bench-all-cumulus.sh#L26-L34 but I dont know what is your use case here :)

**Review by bkontur — APPROVED** [review]

File: substrate/utils/frame/benchmarking-cli/src/pallet/command.rs (hunk 1 above)
Comment (ggwpez) [reply substrate/utils/frame/benchmarking-cli/src/pallet/command.rs:759]: Yea that also has a `uniq` in the pipe to dedub the instances, but now we already do it here.

**Pull request merged**